### PR TITLE
Add Amazing Marvin

### PR DESCRIPTION
**Pull request by rakleed**

### CODE REVIEW

File: entries/a/amazingmarvin.com.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "Amazing Marvin": {
+      "domain": "amazingmarvin.com",
```
Comment (phallobst) [inline entries/a/amazingmarvin.com.json:3]:
```suggestion
      "domain": "amazingmarvin.com",
      "img": "amazingmarvin.com.png",
```

File: entries/a/amazingmarvin.com.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://help.amazingmarvin.com/en/articles/5019430-two-factor-authentication",
```
Comment (phallobst) [inline entries/a/amazingmarvin.com.json]:
```suggestion
      "documentation": "https://help.amazingmarvin.com/en/articles/5019430",
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: entries/a/amazingmarvin.com.json (hunk 2 above)
Comment (rakleed) [reply entries/a/amazingmarvin.com.json]: @phallobst what's the point of shortening the link?

Comment (Carlgo11) [reply entries/a/amazingmarvin.com.json]: @rakleed less bytes with same result.

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**